srml/system/src/lib.rs

```diff
@@ -174,7 +178,8 @@ pub trait Trait: 'static + Eq + Clone {
 	/// Used to define the type and conversion mechanism for referencing accounts in transactions. It's perfectly
 	/// reasonable for this to be an identity conversion (with the source type being `AccountId`), but other modules
 	/// (e.g. Indices module) may provide more functional/efficient alternatives.
-	type Lookup: StaticLookup<Target = Self::AccountId>;
+	// TODO: avoid &'static str error type
+	type Lookup: StaticLookup<Target = Self::AccountId, Error = &'static str>;
```

Comment:
Shouldn't this be `Error=Self::Error`?

Reply:
Yes it should be. But that means a lot more changes in many places. I want to limit the scope of this PR as it is already large. I have already added a TODO and will make an issue to track it.

Reply:
#2953

Reply:
Please add the issue number also to the `TODO` in code.

Reply:
Added